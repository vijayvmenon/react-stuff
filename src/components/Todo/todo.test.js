import React from 'react';
import { shallow } from 'enzyme';
import ToDoApp from "./todo";

const add = (a,b) => a+b;

test('should add two numbers',() => {
    const sum = add(3,4);
    expect(sum).toBe(7);
});

test("it should render Todo Component",() => {
    const wrapper = shallow(<ToDoApp />);
    expect(wrapper).toMatchSnapshot();
})