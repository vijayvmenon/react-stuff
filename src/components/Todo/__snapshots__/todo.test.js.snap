// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render Todo Component 1`] = `
<div>
  To do App
  <WithStyles(ForwardRef(TextField))
    label="Enter Item"
    onChange={[Function]}
    value=""
  />
  <WithStyles(ForwardRef(Button))
    onClick={[Function]}
  >
    Add
  </WithStyles(ForwardRef(Button))>
  <WithStyles(ForwardRef(Button))
    onClick={[Function]}
  >
    Clear List!
  </WithStyles(ForwardRef(Button))>
  <List
    data={Object {}}
  />
</div>
`;
